src/main/app/case/definition.ts

```diff
@@ -433,6 +433,11 @@ export interface CaseData {
   selectedSiblingId: string;
   selectedSiblingPoId: string;
 
+  applicant1StatementOfTruth: YesOrNo;
```

Comment:
Type should be Checkbox

Reply:
Checkbox class is defined in case.ts and applicant1StatementOfTruth: YesOrNo; defined in definition.ts as per other application